responsibleai/responsibleai/managers/error_analysis_manager.py

```diff
@@ -202,19 +172,25 @@ def __init__(self, model, dataset, target_column, classes=None,
         :type classes: list
         :param categorical_features: The categorical feature names.
         :type categorical_features: list[str]
+        :param metadata_columns: The set of columns that are not passed
```

Comment:
Can you elaborate more on what other analysis is intended?

Reply:
Added a bit more context.